### PR TITLE
Use default twilio style. Remove custom font overrides.

### PR DESCRIPTION
**Pull request by yunuscanemre**
Screenshot 2023-03-31 at 11 58 28 AM

### CODE REVIEW

File: src/components/conversations/ConversationContainer.tsx [hunk 1]
```diff
@@ -73,7 +73,6 @@ const ConversationContainer: React.FC<ConvoContainerProps> = (
               flexDirection: "column",
               justifyContent: "center",
               textAlign: "center",
-              fontFamily: "Inter",
```
Comment (yunuscanemre) [inline src/components/conversations/ConversationContainer.tsx:76]: remove all custom overrides of font styles so they update automatically with paste.

File: src/index.tsx [hunk 2]
```diff
@@ -12,14 +12,8 @@ ReactDOM.render(
   <Box style={styles.app}>
     <React.StrictMode>
       <Provider store={store}>
-        <Theme.Provider theme="default">
+        <Theme.Provider theme="twilio">
```
Comment (yunuscanemre) [inline src/index.tsx:15]: twilio theme brings in twilio sans font without needing to manually set it anywhere.

File: src/index.tsx [hunk 3]
```diff
@@ -12,14 +12,8 @@ ReactDOM.render(
...
           <Box style={styles.app}>
-            <link rel="preconnect" href="https://fonts.googleapis.com" />
-            <link rel="preconnect" href="https://fonts.gstatic.com" />
-            <link
-              href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
-              rel="stylesheet"
-            ></link>
```
Comment (yunuscanemre) [inline src/index.tsx:22]: no need to download these.

File: src/index.tsx (hunk 2 above)
Comment (rziehl-twilio) [reply src/index.tsx:15]: This makes me so happy, I need a better gif than the chef kiss one I always use

File: src/index.tsx (hunk 3 above)
Comment (rziehl-twilio) [reply src/index.tsx:22]: Beautiful

**Review by rziehl-twilio — APPROVED** [review]

**Pull request merged**